src/style/colors.ts

```diff
@@ -129,7 +130,8 @@ export class Colors {
       blue = validateRGB(p3!);
       opacity = p4;
     } else {
-      throw new Error('rgba can work with either 2 or 4 arguments');
+      LogService.error('Colors.rgba fail due to invalid arguments');
```

Comment:
I think it should be `failed` (also in tests)

Reply:
What do you mean should be failed? 
LogService.error doesn't throw an error